### PR TITLE
Allow to create deploy token with all possible scopes.

### PR DESCRIPTION
**Pull request by jindrichskupa**
Resolves: https://github.com/gitlabhq/terraform-provider-gitlab/issues/704

Add all existing scopes to provider: https://docs.gitlab.com/ee/api/deploy_tokens.html#create-a-project-deploy-token

### CODE REVIEW

File: gitlab/resource_gitlab_deploy_token.go [hunk 1]
```diff
@@ -176,6 +184,19 @@ func resourceGitlabDeployTokenRead(d *schema.ResourceData, meta interface{}) err
 				if scope == "read_registry" {
 					d.Set("scopes.read_registry", true)
 				}
+
```
Comment (mkjmdski) [inline gitlab/resource_gitlab_deploy_token.go:187]: I think at this point scope should be done as `switch` not a set of `if` conditions

Comment (jindrichskupa) [reply gitlab/resource_gitlab_deploy_token.go]: It could be since begining. But fixed.

**Comment by mkjmdski** [comment]
There is no write repository scope in gitlab API. Must be at least one of read_repository, read_registry, write_registry, read_package_registry, or write_package_registry

**Review by johannges — CHANGES_REQUESTED** [review]
please remove write_repository, because it is not supported.

**Comment by johannges** [comment]
i just opened a pr #769 for this issue, because this pr #695 has not been updated for a while.

**Comment by johannges** [comment]
i just opened a pr #769 for this issue, because the pr #695 has not been updated for a while.

**Comment by timofurrer** [comment]
Since #769 implemented what has been the initial intention of this PR, I'm going to close this one.

**Pull request closed without merging**